### ISSUE TITLE
Won't build on Windows 64bit

### ISSUE THREAD

**Issue by dukc** [mysql-d/mysql-native#194]
On windows, try to initialize a DUB project, and define `dub.sdl`:

```
name "testproject"
description "A minimal D application"
authors "Ate Eskola"
copyright "whoever"
dependency "mysql-native" version="~>2.3"
license "boost"
```

Invoke dub (version 1.14.0): `dub build --arch=x86_64`. Result: `Could not resolve configuration for package testproject`

Removing either `--arch=x86_64` or mysql dependency will make the build succeed.

My symphaties, this looks like something very unexcepted.

**Comment by Abscissa** [comment]
This looks like it's probably a dub issue, I've filed a ticket there: https://github.com/dlang/dub/issues/1713

It looks like there's a newer dub now, v1.15. Could you see if the problem still occurs with that? I don't have a Win development environment on-hand to check with at the moment.

**Comment by dukc** [comment]
> It looks like there's a newer dub now, v1.15. Could you see if the problem still occurs with that?

Yes it does. And also on 1.13, which happens to be verson of my LDC-bundled DUB.

**Comment by schveiguy** [comment]
Not sure if I will fix this. Are you OK upgrading to 3.0.0?

**Comment by dukc** [comment]
No need for me to update anymore:

1. Since my attempts to get SQL working took so long (it wasn't all fault of this issue - I had other problems too, can't remember precisely what tho) we decided to resort to a workaround that does not involve databases.

2. I switched to Linux shortly after my last comment, I haven't built my server for Windows in ages.

3. The project has progressed heavily since I tried to do databases -It would probably take me days to get a test suite again, even with version control. It wasn't exactly the build reproductibility wonder back then.

Thanks for looking at it anyway, whether it's 2.x.x or 3.0.0.
